cmd/get/task.go

```diff
@@ -17,20 +17,26 @@ const (
 	taskShort = "Gets task resources"
 	taskLong  = `
 Retrieves all the task within project and domain.(task,tasks can be used interchangeably in these commands)
+
 ::
 
  bin/flytectl get task -p flytesnacks -d development
 
-Retrieves task by name within project and domain.
 
+Retrieves task by name within project and domain.
 ::
 
  bin/flytectl task -p flytesnacks -d development core.basic.lp.greet
 
-Retrieves project by filters.
+Retrieves all the tasks with filters.
 ::
-
- Not yet implemented
+ 
+ bin/flytectl get task -p flytesnacks -d development core.basic.lp.greet --filters="eq(task.name, "core.basic.lp.greet")" 
+ 
+Retrieves all the task with limit and sorting.
+::
+  
+ bin/flytectl get -p flytesnacks -d development task core.basic.lp.greet  --sort-by=created_at --limit=1 --asc
```

Comment:
can we sort by other fields?

Reply:
yes need to pas field name

Reply:
Can we document which all fields are available to sort and filter and if it too exhaustive then a mechanism in how the user can infer the exact attribute to use for it
This will also help when we add new fields which can potentially be used for sorting or filtering